confidential-data-hub/golang/cmd/grpc-client/main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2024 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	common "github.com/confidential-containers/guest-components/confidential-data-hub/golang/pkg/core"
+	cdhgrpcapi "github.com/confidential-containers/guest-components/confidential-data-hub/golang/pkg/grpc"
+)
+
+func main() {
+
+	vFlag := flag.String("v", "", "sealed secret value")
+	fFlag := flag.String("f", "", "sealed secret file")
```

Comment:
So the client now only supports unseal secret, right? We might need to add some TODO notes inside the code for future extension.

btw, we might also want to add two parameters:
1. GRPC socket address
2. to tell whether this is a gRPC or ttRPC client

Reply:
Done.